frontend/package.json

```diff
@@ -6,10 +6,11 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "start:port": "PORT=3001 next dev"
```

Comment:
This is an extra config to start development os a different port. Used for testing and debugging